Git-commit: 37995f034ff20933f1da12203753a5e6957813d7
Message-Id: <37995f034ff20933f1da12203753a5e6957813d7.1657695708.git.geliang.tang@suse.com>
In-Reply-To: <6c32496964da0dc230cea763a0e934b2e02dabd5.1657695708.git.geliang.tang@suse.com>
References: <6c32496964da0dc230cea763a0e934b2e02dabd5.1657695708.git.geliang.tang@suse.com>
From: Samuel Holland <samuel@sholland.org>
Date: Fri, 1 Jul 2022 22:19:02 -0500
Subject: [PATCH] dt-bindings: dma: allwinner,sun50i-a64-dma: Fix min/max typo
References: bsn#19
Patch-mainline: v5.10.130

commit 607a48c78e6b427b0b684d24e61c19e846ad65d6 upstream.

The conditional block for variants with a second clock should have set
minItems, not maxItems, which was already 2. Since clock-names requires
two items, this typo should not have caused any problems.

Fixes: edd14218bd66 ("dt-bindings: dmaengine: Convert Allwinner A31 and A64 DMA to a schema")
Signed-off-by: Samuel Holland <samuel@sholland.org>
Reviewed-by: Rob Herring <robh@kernel.org>
Link: https://lore.kernel.org/r/20220702031903.21703-1-samuel@sholland.org
Signed-off-by: Vinod Koul <vkoul@kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 .../devicetree/bindings/dma/allwinner,sun50i-a64-dma.yaml       | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/dma/allwinner,sun50i-a64-dma.yaml b/Documentation/devicetree/bindings/dma/allwinner,sun50i-a64-dma.yaml
index 372679dbd216..7e250ce136ee 100644
--- a/Documentation/devicetree/bindings/dma/allwinner,sun50i-a64-dma.yaml
+++ b/Documentation/devicetree/bindings/dma/allwinner,sun50i-a64-dma.yaml
@@ -61,7 +61,7 @@ if:
 then:
   properties:
     clocks:
-      maxItems: 2
+      minItems: 2
 
   required:
     - clock-names
-- 
2.34.1

